lite-locals.tf

```diff
@@ -385,15 +547,16 @@ locals {
     maxGraphs           = 2
     maxGraphStartupTime = 172800
     numWorkersPerPod    = 1
+    #nodeSelector        = {}
     nodeSelector        = { workload = "thorpool" }
     egress              = "engineEgress"
     tolerations_value   = "thorpool"
     managerResources = {
-      cpu    = 2
-      memory = "4G"
+      cpu    = 1
+      memory = "2G"
     }
     workerResources = {
-      cpu    = 4
+      cpu    = 3
```

Comment:
Is 3 the best number here (I thought it was 2).

Reply:
Now, both cpu and memory for 'workerResources' are variable.